components/data_stores/actions/get-record-keys/get-record-keys.mjs

```diff
@@ -0,0 +1,133 @@
+import app from "../../data_stores.app.mjs";
+
+export default {
+  key: "data_stores-get-record-keys",
+  name: "Get Record Keys",
+  description: "Get all record keys in your [Pipedream Data Store](https://pipedream.com/data-stores/) that mathces with your query. The memory consumption of the workflow can be affected, since this action will be exposing, to the workflow, the entire data from the selected datastore",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    dataStore: {
+      propDefinition: [
+        app,
+        "dataStore",
+      ],
+    },
+    query: {
+      type: "string",
+      label: "Query",
+      description: "The query to filter the records.",
+    },
+    contains: {
+      type: "boolean",
+      label: "Contains",
+      description: "If `true`, the query will be used to filter the records that contains the query. If `false`, the query will be used to filter the records that are equal to the query.",
+      optional: true,
+    },
+    caseInsensitive: {
```

Comment:
```suggestion
    caseSensitive: {
```

Optional, but highly recommended. [See this comment.](https://github.com/PipedreamHQ/pipedream/pull/5314#discussion_r1090873831)